packages/network/src/chart/Network.jsx

```diff
@@ -0,0 +1,146 @@
+/* eslint class-methods-use-this: 0, react/no-unused-prop-types: 0 */
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { Group } from '@vx/group';
+import { themeShape } from '../utils/propShapes';
+import WithTooltip, { withTooltipPropTypes } from '../enhancer/WithTooltip';
+import Layout from '../layout/atlasForce';
+import Links from './Links';
+import Nodes from './Nodes';
+import Link from './Link';
+import Node from './Node';
+
+export const propTypes = {
+  ...withTooltipPropTypes,
+  ariaLabel: PropTypes.string.isRequired,
+  width: PropTypes.number.isRequired,
+  height: PropTypes.number.isRequired,
+  graph: PropTypes.shape({
+    nodes: PropTypes.array.isRequired,
+    links: PropTypes.array.isRequired,
+  }).isRequired,
+  layout: PropTypes.string,
+  theme: themeShape,
+  renderTooltip: PropTypes.func,
+  animated: PropTypes.bool,
+};
+
+const defaultProps = {
+  layout: 'none',
+  renderTooltip: null,
+  theme: {},
+  animated: false,
+};
+
+class Network extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      graph: this.copyGraph(props.graph),
+    };
+    this.layout = new Layout({ animated: props.animated });
+    this.layout.setGraph(props.graph);
+    this.layout.layout({
+      callback: (newGraph) => {
+        this.setStateGraph(newGraph);
+      },
+    });
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { graph, animated } = nextProps;
+    if (this.props.graph.links !== graph.links ||
```

Comment:
style wise I would make this
`(this.props.graph.links !== graph.links || this.props.graph.nodes !== graph.nodes)` 
or
```
if (
  this.props.graph.nodes !== graph.nodes ||
  this.props.graph.nodes !== graph.nodes
) { ... }
```